### ISSUE TITLE
Find place for infectiousness calculations

### ISSUE THREAD

**Issue by edwenger** [edwenger/emodlib#7]
Infectiousness is a function of mature gametocyte density, cytokines, and parameters governing sigmoidal relationship.  Existing code is somewhat awkwardly entangled between Susceptibility and Individual objects.  Might be nice to have that code separate from clean intrahost API.

**Comment by edwenger** [comment]
A pure Python function (outside the C++ code) would be fine here -- taking the fever (back-converted to cytokine level) + gametocyte values from the exported C++ functions.

Something like the following from [emod-demo/seasonal_challenge.ipynb](https://github.com/edwenger/emod-demo/blob/master/examples/seasonal_challenge.ipynb):

```
import numpy as np

MICROLITERS_PER_BLOODMEAL = 2.0  # from EMOD
base_gametocyte_mosquito_survival = 0.002011099  # from idmtools-calibra (post-2017 settings)

def infectiousness(g):
    # TODO: add cytokine killing fever effect here also
    return 1-np.exp(-(g * MICROLITERS_PER_BLOODMEAL * base_gametocyte_mosquito_survival))
```

**Comment by edwenger** [comment]
One way to do this is just by attaching a new function in Python after exporting the C++ code: https://github.com/edwenger/emodlib/compare/master...infectiousness#diff-bf1608bfea98913ef9d1587ff901b214fe254b785c1f86a5485485faea6c14ffR17

Which is perhaps a nice experiment in the value of flexibly adjusting functional forms without rebuilding the core package... but it also exposes the fact that the python dictionary of config parameters is parsed to static variables of the C++ objects in a way that's not synchronized between the two sides of that handoff

**Comment by edwenger** [comment]
Pulling into IntrahostComponent C++ code and exporting property as for other variables: https://github.com/edwenger/emodlib/pull/28/commits/da8801469777516ff6d17c85f971f877a0c95d2e

**Comment by edwenger** [comment]
Merged in #28
